fdbt-dev/data/netexData/multiOpFlatFareGeoZone.xml

```diff
@@ -0,0 +1,1906 @@
+<PublicationDelivery xmlns="http://www.netex.org.uk/netex" xmlns:siri="http://www.siri.org.uk/siri" xmlns:gml="http://www.opengis.net/gml/3.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.1" xsi:schemaLocation="http://www.netex.org.uk/netex http://netex.uk/netex/schema/1.10/xsd/NeTEx_publication.xsd">
+  <PublicationTimestamp>2021-07-08T14:03:29.685Z</PublicationTimestamp>
+  <ParticipantRef>SYS001</ParticipantRef>
+  <PublicationRequest version="1.0">
+    <RequestTimestamp>2021-07-08T14:03:29.685Z</RequestTimestamp>
+    <ParticipantRef>SYS002</ParticipantRef>
+    <Description>Request for BLAC bus pass fares</Description>
+    <topics>
+      <NetworkFrameTopic>
+        <selectionValidityConditions>
+          <AvailabilityCondition version="any" id="r1">
+            <FromDate>2017-01-01T00:00:00Z</FromDate>
+          </AvailabilityCondition>
+        </selectionValidityConditions>
+        <TypeOfFrameRef version="fxc:v1.0" ref="fxc:UK:DFT:TypeOfFrame_UK_PI_NETWORK_FARE_OFFER:FXCP"/>
+        <NetworkFilterByValue>
+          <objectReferences>
+            <TypeOfFareProductRef version="fxc:v1.0" ref="fxc:standard_product@pass@period"/>
+            <PreassignedFareProductRef version="1.0" ref="op:Pass@JTest00_adult"/>
+            <BrandingRef version="1.0" ref="op:BLAC@brand"/>
+            <GroupOfOperatorsRef version="1.0" ref="operators@bus"/>
+          </objectReferences>
+        </NetworkFilterByValue>
+      </NetworkFrameTopic>
+    </topics>
+    <RequestPolicy>
+      <RequestDetail>AllWithXref</RequestDetail>
+    </RequestPolicy>
+  </PublicationRequest>
+  <dataObjects>
+    <CompositeFrame version="1.0" id="epd:UK:BLAC:CompositeFrame_UK_PI_NETWORK_FARE_OFFER:Pass@BLAC_products:op" dataSourceRef="op:src" responsibilitySetRef="op:tariffs">
+      <ValidBetween>
+        <FromDate>2002-02-01T00:00:00.000Z</FromDate>
+      </ValidBetween>
+      <Name>Fares for Blackpool Transport</Name>
+      <Description>Period ticket for Blackpool Transport</Description>
```

Comment:
![Screenshot 2023-01-31 at 13 05 25](https://user-images.githubusercontent.com/111580926/215767747-8ea91fcf-9388-43d4-b03b-cac61a65bcfd.png)

Screenshot above is from the netex generator - I dont think we should be saying 'Period ticket'